GVFS/GVFS.Common/NuGetUpgrader/InstallManifest.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections.Generic;
+
+namespace GVFS.Common

```

Comment:
Should this all be in `GVFS.Common` or something like `GVFS.Common.Upgrade`?

Reply:
And for file locations: Should we have an `Upgrade` folder with a `NuGet` folder inside?

(I see that we have a lot of upgrader code in the base of the project, which I wish I noticed earlier. It would be better to group those things into a folder.)

Reply:
The namespace has been updated to match the folder structure.

The upgrader files could probably be more organized. Nesting too deeply comes with its own set of downsides. Maybe a good compromise would be 3 top level folders: `Upgrader`, `GitHubUpgrader`, and `NuGetUprader`, or maybe a single top level directory of `Upgrader`, or maybe we do add two levels of nesting...